flux/src/checkflux/index.tsx

```diff
@@ -43,11 +28,9 @@ export default function CheckIfFluxInstalled() {
         </p>
       </Box>
     );
-  }
 }
 
-export function useFluxControllerAvailableCheck(props: { name: string }) {
-  const { name } = props;
+export function useFluxInstallCheck() {
   const [deployments] = K8s.ResourceClasses.Deployment.useList({
     labelSelector: 'app.kubernetes.io/part-of=flux',
```

Comment:
Seems like you could use label selector with both labels here, then `deployments?.find(...` won't be neccessary?
```tsx
labelSelector: 'app.kubernetes.io/part-of=flux,app.kubernetes.io/component=source-controller',
```

Reply:
Good point!